app/controllers/Api2.scala

```diff
@@ -70,69 +70,47 @@ class Api2 @Inject() (implicit val previewDataStore: PreviewDataStore,
 
   def publishMediaAtom(id: String) = APIHMACAuthAction { implicit req =>
     implicit val user = req.user
-    try {
-      val updatedAtom = PublishAtomCommand(id).process()
-      Ok(Json.toJson(updatedAtom))
-    } catch {
-      commandExceptionAsResult
-    }
+
+    val updatedAtom = PublishAtomCommand(id).process()
+    Ok(Json.toJson(updatedAtom))
```

Comment:
as we've removed the `try/catch` what happens when `PublishAtomCommand(id).process()` do we get a 500 and a sensible message still?